Product/Production/Services/PatientDiscoveryCore/src/test/java/gov/hhs/fha/nhinc/patientdiscovery/inbound/deferred/response/PassthroughInboundPatientDiscoveryDeferredResponseTest.java

```diff
@@ -54,11 +52,14 @@ public void invoke() {
         PRPAIN201306UV02 request = new PRPAIN201306UV02();
         AssertionType assertion = new AssertionType();
         MCCIIN000002UV01 expectedResponse = new MCCIIN000002UV01();
+        NhinTargetSystemType target = new NhinTargetSystemType();
 
         // Mocks
-        AdapterPatientDiscoveryDeferredRespProxyObjectFactory adapterFactory = mock(AdapterPatientDiscoveryDeferredRespProxyObjectFactory.class);
+        AdapterPatientDiscoveryDeferredRespProxyObjectFactory adapterFactory = 
+            mock(AdapterPatientDiscoveryDeferredRespProxyObjectFactory.class);
         AdapterPatientDiscoveryDeferredRespProxy adapterProxy = mock(AdapterPatientDiscoveryDeferredRespProxy.class);
-        PatientDiscoveryAuditor auditLogger = mock(PatientDiscoveryAuditor.class);
+        PatientDiscoveryDeferredResponseAuditLogger auditLogger = mock(PatientDiscoveryDeferredResponseAuditLogger.class);
```

Comment:
Don't forget to line wrap when breaking past 120 chars; there will always be exceptions (NB auto formatter dislikes breaking up a long diamond operator, and it has some weird rules around multi-param annotations), but in general we should be trying to wrap for readability.